src/main/scala/fr/polytechnique/cmap/cnam/etl/extractors/mco/McoExtractor.scala

```diff
@@ -32,19 +32,43 @@ trait McoExtractor[EventType <: AnyEvent] extends Extractor[EventType] with McoS
     Seq(eventBuilder[EventType](patientId, groupId, code(row), weight, eventDate, endDate))
   }
 
+  /** It gets columnName's value from row.
+   *
+   * @return A string value.
+   */
   def code = (row: Row) => row.getAs[String](columnName)
 
+  /** It gets PatientID value from row.
+   *
+   * @param r The row itself.
+   * @return The value of PatientID.
+   */
   def extractPatientId(r: Row): String = {
     r.getAs[String](ColNames.PatientID)
   }
 
+  /** It gets EtaNum value, RsaNum value and Year from row.
```

Comment:
This is wrong description. This is specific to Mco Source and how we group Events from this source together. Better phrasing : 

_Creates an ID that group Events of different categories by concatinating ETA_NUM, RSA_NUM and the YEAR._